bundles/http/Citrix/AdobeSign/file.sapp

```diff
@@ -9301,7 +9304,7 @@
                 "style" : null,
                 "to" : null,
                 "serviceId" : null,
-                "value" : "https://secure.in1.echosign.com/public/agreements/view/CBJCHBCAABAA-9G3p4XHR6_S3SvpsSRSR_4lxCS2XThb?type=esign",
+                "value" : "https://.in1.adobesign.com/public/agreements/view/CBJCHBCAABAA-2lNQ_G0TiBl45XgOIKuHFjgNpDavDzN?type=esign",
```

Comment:
Isn't the {yourAbodeSignHostname} missing? And should't the ID be masked too?

Reply:
changes done